src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md

```diff
@@ -35,15 +35,11 @@ For example when using the [`#if` preprocessor directive](https://docs.microsoft
 
 To correctly model the API differences between target frameworks (or any other property), use multiple instances of the `PublicAPI.*.txt` files.
 
-For example, if you target both `net4.8` and `netcoreapp3.0` target frameworks, and APIs differ between each, then you would have the following:
+For example, if you target both `net48` and `netcoreapp3.0` target frameworks, and APIs differ between each, then you would have the following:
 
 ```xml
-  <ItemGroup Condition="'$(TargetFramework)' == 'net4.8'">
-    <AdditionalFiles Include="net4.8/PublicAPI.Shipped.txt" />
-    <AdditionalFiles Include="net4.8/PublicAPI.Unshipped.txt" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-    <AdditionalFiles Include="netcoreapp3.0/PublicAPI.Shipped.txt" />
-    <AdditionalFiles Include="netcoreapp3.0/PublicAPI.Unshipped.txt" />
+  <ItemGroup>
+    <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
+    <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Unshipped.txt" />
   </ItemGroup>
```

Comment:
📝 The Properties window in VS 16.6-p2 seems to struggle equally with both this form and the original form, showing only the files associated with the first target framework as AdditionalFiles and the rest as None. PublicApiAnalyzers itself works equally well with both forms.

Reply:
Can you please file an issue in https://github.com/dotnet/project-system repo for the properties window? Thanks for your help!

Reply:
Sure! https://github.com/dotnet/project-system/issues/6042